GoogleForCreators/web-stories-wp
# Karma: Add snapshot testing to dashboard

## Context

Adds snapshot testing to the dashboard

See https://github.com/GoogleForCreators/web-stories-wp/issues/10392#issue-1115301175

## Summary

- Updates tests-karma-dashboard.yml workflow to allow capturing snapshots
- Adds some initial snapshots

## Relevant Technical Choices

- Updated Karma fixture to return `allPagesFetched: true` see:  https://github.com/GoogleForCreators/web-stories-wp/pull/10671#discussion_r816328177

- Removes Karma banner see https://github.com/GoogleForCreators/web-stories-wp/pull/10671/files#diff-2594fcd692a91cd97c36d0083ba11e89c09010cc0b2a8f26563b76d87503d621L108

## To-do



## User-facing changes

N / A

## Testing Instructions

To view / ensure snapshots are working

Check the Dashboard workflow i.e. 
https://github.com/GoogleForCreators/web-stories-wp/runs/5368291108?check_suite_focus=true

https://percy.io/google/web-stories-wp-dashboard/builds/16128639





## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

Adds more screenshots to end to end tests

### Does this PR have a legal-related impact?

No

## Checklist




---



Fixes #10392
